### ISSUE TITLE
"Expect troubles !" error from `create_window`

### ISSUE THREAD

**Issue by bryankaplan** [r0x0r/pywebview#1457]
# Specification
- pywebview version: 5.2
- operating system: Void Linux
- web renderer: qt

# Description

## Overview

Calling `webview.create_window` from a function causes the following error:

> Release of profile requested but WebEnginePage still not deleted. Expect troubles !

## Steps to Reproduce

1. `$ python -m venv venv`
2. `$ . ./venv/bin/activate`
3. `$ pip install 'pywebview[qt]'`
4. Write test.py as follows:
```python
import webview

def go(title, url):
    webview.create_window(title, url)

if __name__ == '__main__':
    go('one', 'http://example.com/')
    webview.start(go, ('two', 'http://example.com/'), gui='qt')
```
5. `$ python -m test`
6. Close both of the opened test windows.

## Expected Result

No error message appears.

## Observed Result

A message is printed to stderr:

> Release of profile requested but WebEnginePage still not deleted. Expect troubles !

## Related

Cf. 38462445f9e23d0b91084baa82ebb5572a760d2c

# Practicalities

* [x] I am willing to work on this issue myself.
* [ ] I am prepared to support this issue financially.

**Comment by bryankaplan** [comment]
I should clarify that while I am indeed willing to work on this issue myself, I don't know where to start, so I could use some guidance.

**Comment by bryankaplan** [comment]
N.B. The [Multiple Windows](https://pywebview.flowrl.com/examples/multiple_windows.html) example demonstrates this same bug.

**Comment by r0x0r** [comment]
I have investigated this issue and ended up adding `self.view.page().deleteLater()` in the close event. I am not sure if this is a correct way to solve this problem, but it makes this warning go away. Improvements are welcomed.
The logic did not work correctly for multiple windows, but it is fixed now in master.

**Comment by bryankaplan** [comment]
Thank you. Based on what you wrote, I confirmed that the following is warning-free:

```python
import webview

def go(title, url):
    window = webview.create_window(title, url)
    window.events.shown += prepare_cleanup

def prepare_cleanup(window):
    browser_view = window.gui.BrowserView.instances.get(window.uid)
    window.events.closed += lambda: browser_view.view.page().deleteLater()

if __name__ == '__main__':
    window = go('one', 'http://example.com/')
    webview.start(go, ('two', 'http://example.com/'), gui='qt')
```
I'm not sure if there's a simpler way to do it. But if it's fixed in master, then the workaround is temporary anyway. Which commit introduced the fix?

As for whether or not it's the correct solution, I have no idea, but it seems reasonable to me.

**Comment by r0x0r** [comment]
Sorry, the previous push failed. The commit is pushed now.

**Comment by bryankaplan** [comment]
Thank you. For anyone else reading, the fix is 5694b6babfc08cf754d6ad6efbb5bc1c059915c2. Issue resolved.
